doc/_utils/redirects.py

```diff
@@ -203,31 +203,6 @@
     ("app_dev/device_guides/working_with_nrf/nrf54l/nrf54l15_gs", "gsg_guides"),
```

Comment:
Also, tag file would require an update - nrf\doc\_zoomin\ncs.tags

Reply:
Yes, the line here needs to be updated with the correct path: https://github.com/nrfconnect/sdk-nrf/blob/main/doc/_zoomin/ncs.tags.yml#L209

Reply:
updated